common/utils/localStorage.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+export const REDUX_STATE = 'REDUX_STATE';
+
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem(REDUX_STATE);
+    if (serializedState === null) {
+      return undefined;
+    }
+    return JSON.parse(serializedState);
+  } catch (err) {
+    // TODO - log warning? or bubble?
```

Comment:
we can console.log('Warn: blah')

dont think user cares much about corrupted localStorage